packages/lesswrong/server/cron/allCronJobs.ts

```diff
@@ -1,6 +1,8 @@
 import { type CronJobSpec } from "@/server/cron/cronUtil";
 import { getAllPostgresViews } from "../postgresView";
+import { reviewVoteTotalsCronJob } from "./reviewVoteTotalsCron";
 
 export const allCronJobs: (CronJobSpec|null)[] = [
   ...getAllPostgresViews().map((view) => view.getCronJob()),
+  reviewVoteTotalsCronJob,
```

Comment:
This is not how you add a cron job anymore (we should delete this code, but doesn't need to be in this PR).  See `vercel.json` and all of the corresponding `api/cron/*` routes